### ISSUE TITLE
Potential javax.xml.bind.PropertyException warning

### ISSUE THREAD

**Issue by shimkiv** [allure-framework/allure-core#541]
It seems that [AllureResultsUtils.marshaller](https://github.com/allure-framework/allure-core/blob/b0f334a7b046e9d27c9874e7d3758a2ca4ff64eb/allure-java-adaptor-api/src/main/java/ru/yandex/qatools/allure/utils/AllureResultsUtils.java#L168) should use [String](https://docs.oracle.com/javase/7/docs/api/javax/xml/bind/Marshaller.html#supportedProps) instead of [Charset](http://docs.oracle.com/javase/7/docs/api/java/nio/charset/Charset.html). 
Otherwise "javax.xml.bind.PropertyException" can be [thrown](http://grepcode.com/file/repository.grepcode.com/java/root/jdk/openjdk/7u40-b43/javax/xml/bind/helpers/AbstractMarshallerImpl.java#AbstractMarshallerImpl.checkString%28java.lang.String%2Cjava.lang.Object%29):
```
javax.xml.bind.PropertyException: jaxb.encoding must be a String
	at javax.xml.bind.helpers.AbstractMarshallerImpl.checkString(AbstractMarshallerImpl.java:426)
	at javax.xml.bind.helpers.AbstractMarshallerImpl.setProperty(AbstractMarshallerImpl.java:333)
	at com.sun.xml.internal.bind.v2.runtime.MarshallerImpl.setProperty(MarshallerImpl.java:527)
	at ru.yandex.qatools.allure.utils.AllureResultsUtils.setPropertySafely(AllureResultsUtils.java:196)
	at ru.yandex.qatools.allure.utils.AllureResultsUtils.marshaller(AllureResultsUtils.java:168)
	at ru.yandex.qatools.allure.utils.AllureResultsUtils.writeTestSuiteResult(AllureResultsUtils.java:146)
	at ru.yandex.qatools.allure.utils.AllureResultsUtils.writeTestSuiteResult(AllureResultsUtils.java:134)
	at ru.yandex.qatools.allure.Allure.fire(Allure.java:190)
	at ru.yandex.qatools.allure.testng.AllureTestListener.onFinish(AllureTestListener.java:106)
        ...
 Can't set "jaxb.encoding" property to given marshaller
```

**Comment by baev** [comment]
@shimkiv thanks for suggesting! Could you please provide the PR and fix it?

**Comment by shimkiv** [comment]
@baev Done

**Comment by baev** [comment]
closed via #542
